packages/fx-core/src/question/inputs/CreateProjectInputs.ts

```diff
@@ -14,7 +14,12 @@ export interface CreateProjectInputs extends Inputs {
   /** @description Teams Toolkit: select runtime for your app */
   runtime?: "node" | "dotnet";
   /** @description New Project */
-  "project-type"?: "bot-type" | "tab-type" | "me-type" | "outlook-addin-type" | "office-addin-type";
+  "project-type"?:
+    | "bot-type"
+    | "tab-type"
+    | "me-type"
+    | "office-xml-addin-type"
+    | "office-addin-type";
```

Comment:
This file should be reverted. This is static helper for CLI. In principle, the static options should not be sensitive to environmental variables: 
<img width="776" alt="image" src="https://github.com/OfficeDev/TeamsFx/assets/1658418/985ff68c-93bd-4cf5-870e-b60e2c295b6c">
This file is automatically generated by `npm run build` in fx-core package, which it will generate different content in inputs/xxxInputs.ts files. This is not expected.

Reply:
Expected: return three option values (office-addin-type, outlook-addin-type, office-xml-addin-type) as static options.

Reply:
DONE